src/teams.rs

```diff
@@ -54,6 +78,15 @@ pub struct FcpBehavior {
     postpone: bool,
 }
 
+#[derive(Debug, Deserialize)]
+#[serde(untagged)]
+enum RfcbotTeams {
+    Local(BTreeMap<TeamLabel, Team>),
+    Remote {
+        url: String,
```

Comment:
Also why the mix of tuple/braced? Pick one?

Reply:
This is mixed because it's the cleanest way to get the deserialization I want with serde. With this `#[serde(untagged)]` enum it accepts either:

```toml
[teams]
url = "http://example.com"
# or
[teams]
[teams.T-infra]
...
```

To get the same deserialization I'd have to do:

```rust
#[derive(Deserialize)]
#[serde(untagged)]
enum RfcbotTeams {
    Local(TeamMap),
    Remote(RemoteTeams),
}
#[derive(Deserialize)]
struct RemoteTeams {
    url: String,
}
```

*I think* it's better to mix the two styles than to create a struct that's used only for this, but it's your codebase so what do you prefer? :)

Reply:
Ah; interesting. Keep it as-is then. Can you leave a comment saying why it is this way tho?

Reply:
Sure thing.